src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportApprovedEpisodes.cs

```diff
@@ -122,9 +122,34 @@ public List<ImportResult> Import(List<ImportDecision> decisions, bool newDownloa
 
                     _eventAggregator.PublishEvent(new EpisodeImportedEvent(localEpisode, episodeFile, oldFiles, newDownload, downloadClientItem));
                 }
+                catch (DirectoryNotFoundException e)
```

Comment:
It'd be better to throw a specific exception if the root folder can't be found instead of `DirectoryNotFoundException` and then check the exception message. Perhaps a `RootFolderNotFoundException` that inherits `DirectoryNotFoundException`.